benchmarks/Elastic.Transport.Benchmarks/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using BenchmarkDotNet.Running;
+
+namespace Elastic.Transport.Benchmarks
+{
+	internal class Program
+    {
```

Comment:
mixed `tabs` and `spaces`. This should be covered by `.editorconfig`

Reply:
Odd! Yep it should have picked those up. Fixed in new commit.